DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/platform_designer_standard/README.md

```diff
@@ -0,0 +1,360 @@
+# `Platform Designer Standard` Sample
+
+This example design shows how to use an FPGA IP produced with the Intel® oneAPI DPC++/C++ Compiler with the Intel® Quartus® Prime Standard Edition.
+
+| Optimized for                     | Description
+|:---                               |:---
+| OS                                | Ubuntu* 20.04 <br> RHEL*/CentOS* 8 <br> SUSE* 15 <br> Windows* 10 <br> Windows Server* 2019
+| Hardware                          | This process applies to the Cyclone® V line of FPGAs, but the sample Intel® Quartus® Prime Standard Edition project targets the [Terasic DE1-SOC Development Board](https://de1-soc.terasic.com/)
+| Software                          | Intel® oneAPI DPC++/C++ Compiler <br> Intel® Quartus® Prime Standard Edition Version 22.3 or later
+| What you will learn               | How to integrate an RTL IP generated from a SYCL kernel with an Intel® Quartus® Prime Standard Edition
+| Time to complete                  | 1 hour
+
+> **Note**: Even though the Intel DPC++/C++ oneAPI compiler is enough to compile for emulation, generating reports and generating RTL, there are extra software requirements for the simulation flow and FPGA compiles.
+>
+> To use the simulator flow, Intel® Quartus® Prime Standard Edition and one of the following simulators must be installed and accessible through your PATH:
+> - Questa*-Intel® FPGA Edition
+> - Questa*-Intel® FPGA Starter Edition
+> - Questa* Advanced Simulator
+> - ModelSim® SE
+>
+> To use the hardware compile flow, Intel® Quartus® Prime Standard Edition must be installed and accessible through your PATH.
+
+## Prerequisites
+
+This sample is part of the FPGA code samples.
+It is categorized as a Tier 1 sample that helps you getting started.
```

Comment:
If this is a Tier 1, then it is not categorized correctly in the main README, if this is a Tier 3 sample, this section is incorrect
I can see that there is the same issue in the platform_designer sample, so both should be addressed - please synchronize with @whitepau to decide where these samples should live

Reply:
Both Platform Designer samples should both be Tier 1 samples.